modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -90,11 +90,11 @@ data:
       remoteWrite:
       - url: "https://remote-write-endpoint.example.com"
         <endpoint_authentication_credentials>
-        <write_relabel_configs> <1>
+        <write_relabel_configs_placeholder> <1>
```

Comment:
```suggestion
        <your_write_relabel_configs> <1>
```
Maybe this change works to avoid confusion here?

[Here are the docs guidelines for user-replaceable values](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values), which say to use underscores.

Reply:
even better.